src/test/regress/sql/full_join.sql

```diff
@@ -112,4 +115,58 @@ SELECT * FROM
 -- Full join using multiple columns
 SELECT * FROM test_table_1 FULL JOIN test_table_2 USING(id, val1) ORDER BY 1,2;
 
+SET citus.enable_repartition_joins to ON;
+
+INSERT INTO abcd VALUES (1,2,3,4);
+INSERT INTO abcd VALUES (2,3,4,5);
+INSERT INTO abcd VALUES (3,4,5,6);
+
+SELECT * FROM abcd first join abcd second on first.a = second.a ORDER BY 1,2,3,4;
+SELECT * FROM abcd first join abcd second on first.b = second.b ORDER BY 1,2,3,4;
+SELECT * FROM abcd first join abcd second on first.c = second.c ORDER BY 1,2,3,4;
+
+BEGIN;
+SELECT * FROM abcd first join abcd second on first.a = second.a ORDER BY 1,2,3,4;
+SELECT * FROM abcd first join abcd second on first.b = second.b ORDER BY 1,2,3,4;
+SELECT * FROM abcd first join abcd second on first.c = second.c ORDER BY 1,2,3,4;
+END;
+
+ALTER TABLE abcd DROP COLUMN a;
+
+SELECT * FROM abcd first join abcd second on first.b = second.b ORDER BY 1,2,3,4;
+SELECT * FROM abcd first join abcd second on first.c = second.c ORDER BY 1,2,3,4;
+
+BEGIN;
+SELECT * FROM abcd first join abcd second on first.b = second.b ORDER BY 1,2,3,4;
+SELECT * FROM abcd first join abcd second on first.c = second.c ORDER BY 1,2,3,4;
+END;
+
+CREATE VIEW abcd_view AS SELECT * FROM abcd;
+
+SELECT * FROM abcd_view first join abcd_view second on first.b = second.b ORDER BY 1,2,3,4;
+SELECT * FROM abcd_view first join abcd_view second on first.c = second.c ORDER BY 1,2,3,4;
+
+BEGIN;
+SELECT * FROM abcd_view first join abcd_view second on first.b = second.b ORDER BY 1,2,3,4;
```

Comment:
it's a bit odd that most of these are not full joins, given the name of the file.

Reply:
Yeah I know, I actually looked a bit for a file to add joins but couldn't find a better file ( we could also create a new file but it didn't seem necessary when we had this file). Should we maybe rename this file to `join.sql`?

Reply:
that works